javascript/javascript3/week1/json and cool Api/script.js

```diff
@@ -0,0 +1,26 @@
+// Create your own json file
+
+const getMyJson = async () => {
+  const pc = await fetch("data.json");
```

Comment:
You can also use the [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) command to do this. I will show it tomorrow.